src/util/system.h

```diff
@@ -248,6 +251,20 @@ class ArgsManager
      */
     const std::list<SectionInfo> GetUnrecognizedSections() const;
 
+    struct Command {
```

Comment:
in fa61b9d1a68820758f9540653920deaeae6abe79:

nit: seeing this now I think I would have named it `CommandCall` or something similar to express that it's command + args. But feel free to ignore my name bikeshedding.

Reply:
https://github.com/bitcoin/bitcoin/pull/20715#discussion_r557886983 names it `CmdArg`. So I could call it `CommandArg` or `CommandArgs` (because there might be multiple args to the command). Not sure what to do here, but I think I'll leave this as is for now and let the naming committee determine a suitable name and let them change it after merge.